packages/backend-proxy-middleware/test/base/proxy.test.ts

```diff
@@ -367,6 +372,24 @@ describe('proxy', () => {
             expect(options.xfwd).toBeUndefined();
             expect(options.secure).toBeUndefined();
         });
+
+        test('generate proxy middleware options for FLP Embedded flow', async () => {
+            const backend: LocalBackendConfig = {
+                url: 'http://backend.example',
+                path: '/my/path',
+                bsp: 'my_bsp'
+            };
+            const answers = {
+                username: '~user',
+                password: '~password'
+            };
+            mockIsAppStudio.mockReturnValue(false);
+            mockPrompt.mockResolvedValue({ ...answers, authNeeded: true });
+            const options = await generateProxyMiddlewareOptions(backend);
+            expect(options.pathRewrite).toBeDefined();
+            expect(options.router).toBeDefined();
+            expect(options.auth).toBeDefined();
```

Comment:
Is it possible to check for more specific values without a deep coupling to the implementation. I'm wondering if just checking for being defined will give us false positives

Reply:
Yes, done with https://github.com/SAP/open-ux-tools/pull/542/commits/2bd3133185713e7db094aff63065da19dd507a94